controllers/vpcendpoint/helpers.go

```diff
@@ -17,11 +17,69 @@ limitations under the License.
 package vpcendpoint
 
 import (
+	"context"
 	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/aws-sdk-go/service/route53"
 	"github.com/openshift/aws-vpce-operator/api/v1alpha1"
+	"github.com/openshift/aws-vpce-operator/pkg/aws_client"
+	"github.com/openshift/aws-vpce-operator/pkg/dnses"
+	"github.com/openshift/aws-vpce-operator/pkg/infrastructures"
+	"github.com/openshift/aws-vpce-operator/pkg/util"
 )
 
+// parseClusterInfo fills in the ClusterInfo struct values inside the VpcEndpointReconciler
+func (r *VpcEndpointReconciler) parseClusterInfo(ctx context.Context) error {
+	r.ClusterInfo = new(ClusterInfo)
+
+	region, err := infrastructures.GetAWSRegion(ctx, r.Client)
+	if err != nil {
+		return err
+	}
+	r.ClusterInfo.Region = region
+	r.Log.V(1).Info("Parsed region from infrastructure", "region", region)
```

Comment:
Can we start using requestLogger instead of the reconcile logger?

Reply:
Will be done in the next PR